src/main/scala/org/broadinstitute/dsde/snoop/ws/SnoopApiFormats.scala

```diff
@@ -5,6 +5,17 @@ import spray.json._
 import com.wordnik.swagger.annotations.{ApiModel, ApiModelProperty}
 import scala.annotation.meta.field
 
+/**
+ * Marker trait for normal messages
+ */
+trait RestMessage
+
+/**
+ * Message used to signify that a validation error happened
+ * @param message
+ */
+case class Validation(message: String)
```

Comment:
Ack, I saw it in the receive method but missed that it wasn't ever used. I change my comment to the other one. I don't think we should have crud in here just because it was patterned on something else. 

On the second one I was just saying that for instance if I had a FooActor class I like having a sealed trait in its companion object FooActorMessage and all case class/objects that are messages for FooActor extend FooActorMessage